### DISCUSSION TITLE
Add json output

### DISCUSSION THREAD

**Discussion by DeGraciaMathieu** [Halleck45/ast-metrics#30] [Ideas]
Hi,

Could we possibly add a JSON output format for the analysis results ?

This would enable us to interface with the tool. I often use small PHP scripts to retrieve and analyze PHPMetrics results.

**Comment by Halleck45** [comment]
Good idea. It's a good first step anyway for an external contribution. I'll see if I can document the subject enough, otherwise I'll do it.

**Reply by Halleck45** [reply]
FI I've opened [an issue](https://github.com/Halleck45/ast-metrics/issues/45)

**Comment by Halleck45** [comment]
Thanks to @julian776 , this feature is now merged
